.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] It would be nice if you add the following message instead of anything in the Acknowledgement section to make your readme descriptive and professional 👍

`I would like to thank Microverse`

Reply:
thank you @DammyShittu changes Done